README.md

```diff
@@ -52,6 +52,22 @@ application should be configured to be a **Regular Web Application**.
 * `AUTH0_DOMAIN`: The Auth0 application's domain
 * `AUTH0_KEY`: The Auth0 application's client id
 * `AUTH0_SECRET`: The Auth0 application's client secret.
+* `TWILIO_ACCOUNT_SID`: Twilio Account SID for client
+* `TWILIO_AUTH_TOKEN`: Twilio Auth Token for client
+* `TWILIO_SMS_NUMBER`: Phone number received from Twilio for client
+
+## Running NGROK
+NGROK is needed to manually test sending text messages
+
+- In your root directory:
+
+` ./ngrok http 80 `
```

Comment:
perhaps a more normal way to write this is without the `./` since it seems to imply that the tool has to be in some directory. Also, I imagine most people would get `ngrok` from a package manager like Home Brew.